File: examples/key/load_local_certificate/load_local_certificate.py
import os

from bloock.client.key import KeyClient

if __name__ == "__main__":
    key_client = KeyClient()

    current_directory = os.getcwd()
    file_path = current_directory + "certificate.p12"
    with open(file_path, 'rb') as file:
        file_bytes = file.read()

    local_certificate = key_client.load_local_certificate(file_bytes, "password")

File: examples/key/load_local_certificate/load_local_certificate.php
<?php

require "./vendor/autoload.php";

use Bloock\Client\KeyClient;

$keyClient = new KeyClient();

$currentDirectory = getcwd();
$fileContents = file_get_contents($currentDirectory . "certificate.p12");
if ($fileContents !== false) {
  $byteArray = unpack('C*', $fileContents);

  if ($byteArray !== false) {
    $certificate = $keyClient->loadLocalCertificate($byteArray, "password");
  }
}
